lib/v2/fuliba/latest.js

```diff
@@ -0,0 +1,51 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+
+module.exports = async (ctx) => {
+    //    const domain = ctx.query.domain ?? 'fuliba.net';
+    const res = await got(`https://fuliba2023.net`, {
+        headers: {},
+    });
+    const $ = cheerio.load(res.data);
+    const list = $('article');
+    const count = list
+        .slice(0, ctx.query.limit ? Number(ctx.query.limit) : 40)
+        .toArray()
+        .map((each) => {
+            each = $(each);
+            const floor = each.find('header h2').text();
+            const floorUrl = each.find('header a').attr('href');
+            return {
+                title: String(floor),
+                link: String(floorUrl),
+                category: each.find('.meta .cat').text(),
+                pubDate: each.find('.meta time').text(),
+            };
+        });
+
+    const resultItems = await Promise.all(
+        count.map((i) =>
+            ctx.cache.tryGet(i.link, async () => {
+                i.description = await fetchContent(i.link);
+                return i;
+            })
+        )
+    );
```

Comment:
Please use [WordPress REST API](https://developer.wordpress.org/rest-api/) instead.
The API endpoints are available at `/wp-json/wp/v2`.

You can list recent posts using `/wp-json/wp/v2/posts`. If you want categories and tags for each post in the posts API, you can add the `_embed` search parameter.

To adjust the no. of articles returned in the posts API, you can pass the `per_page` search parameter.

Reply:
OK, honestly I even didn't realize that site is built on top of WP....interesting.

I need to take some time to refine this part , maybe update tomorrow. thanks.